src/openpgp/processor.iced

```diff
@@ -60,7 +60,7 @@ class KeyBlock
 
   _check_primary : () ->
     err = if not @primary.is_self_signed()
-      new Error "no valid primary key self-signature"
+      new Error "no valid primary key self-signature or signature(s) have expired"
```

Comment:
well it's not that signature have expired --- it's that the key producing the signatures have expired.